index.html

```diff
@@ -943,17 +974,46 @@ <h4>publicKeyPem</h4>
                 <a href="https://w3id.org/security/#publicKeyPem">security-vocab</a>
               </td>
               <td>
+                <a href="https://www.w3.org/ns/did/v1">DID Core context</a>
+              </td>
+              <td>
+                <a href="https://ipfs.io/ipfs/QmQHYAq7VxrKKU8o6qW7iJPZygLefPa1qMyozVRGBU8yfZ" target="_blank">publicKeyPem.cddl</a>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+
+      </section>
+      <section>
+        <h4>publicKeyGpg</h4>
+
+        <table class="simple" style="width: 100%;">
+          <thead>
+            <tr>
+              <th>Normative Definition</th>
+              <th>JSON-LD</th>
+              <th>CDDL</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <a href="https://w3id.org/security/#publicKeyGpg">security-vocab</a>
```

Comment:
This does not exist in the security vocabulary. It needs to be added before merging this PR.

Reply:
@jonnycrunch just remove this for now, I have opened https://github.com/w3c-ccg/security-vocab/issues/74

Reply:
publicKeyGpg removed but left template for completion.